Samples/Gtk3/Template/CompositeWidget.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Reflection;
+using GObject;
+using Gtk;
+using Type = GObject.Type;
+
+namespace GtkDemo
+{
+    public class CompositeWidget : Bin
+    {
+        private static void ClassInit(Type gClass, System.Type type, IntPtr classData)
+        {
+            SetTemplate(
+                gtype: gClass, 
+                template: Assembly.GetExecutingAssembly().ReadResource("CompositeWidget.ui")
+            );
+            BindTemplateChild(gClass, nameof(Button));
+            ConnectTemplateSignals(gClass, type);
+        }
+
+        protected override void Initialize()
+        {
+            InitTemplate();
+            BindTemplateChild(nameof(Button), ref Button);
```

Comment:
What's the difference between `BindTemplateChild` and `BindTemplateChild<T>`? As we use them both for `Button`, they really should have different names.

Maybe `BindTemplateChildForClass` or `ClassBindTemplateChild` for the static one?

Reply:
I used different names:

- `ConnectTemplateChildToField` I hope this makes it clear that we "Connect" the template to the field. I think connect is a good name because the attribute to connect fields is the `ConnectAttribute`, too.
- In the `static ClassInit` I only use `BindXX` methods as those work on the native / class level.

What do you think of this names?

Reply:
I like this a lot more. Much less ambiguous.